tools/pip/setup.py

```diff
@@ -142,22 +142,16 @@ def skip_markdown_comments(md):
         libraries.append('CUDA-10.0')
     elif variant.startswith('CU92'):
         libraries.append('CUDA-9.2')
-    elif variant.startswith('CU91'):
```

Comment:
Are we dropping support for all cuda till 9? I guess yes [since cuda11 has launched and we support till latest major version-1]. Can we update the PR title to also reflect the same? Since currently it just talks about mkldnn and this PR changes things more than that.